drivers/ethernet/eth_sam_gmac.c

```diff
@@ -174,7 +174,6 @@ BUILD_ASSERT(DT_INST_ENUM_IDX(0, phy_connection_type) <= 1, "Invalid PHY connect
 		DEFN_RX_FRAG(n)								\
 		DEFN_TX_FRAG(n)								\
 		DEFN_TX_FRAME(n)
-DT_INST_FOREACH_STATUS_OKAY(SAM_GMAC_LISTS_DEFN)
```

Comment:
Be careful with size of commit brief, see [pull-request-guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#pull-request-guidelines)